src/db.ts

```diff
@@ -0,0 +1,11 @@
+import { IConnectionParameters } from 'pg-promise/typescript/pg-subset';
+
+const config: Readonly<IConnectionParameters> = {
+  user: process.env.DATABASE_USER,
+  password: process.env.DATABASE_PASSWORD,
+  database: process.env.DATABASE_NAME,
+  host: process.env.DATABASE_HOST,
+  port: process.env.DATABASE_PORT ? parseFloat(process.env.DATABASE_PORT) : undefined,
```

Comment:
I understand there are no integers in JavaScript, but `parseInt` doesn't work? :thinking:

Reply:
Didn’t have time to work out what was going on with eslint. Understand now, but that rule applies to Browsers not Node so would need to disable. Doing the `+` trick elsewhere.